### DISCUSSION TITLE
GitPOAPs for contributors (and/or taskers)

### DISCUSSION THREAD

**Discussion by rasdani** [LAION-AI/Open-Assistant#3430] [Ideas]
A [GitPOAP](https://www.gitpoap.io/) is an [NFT badge](https://www.gitpoap.io/gitpoaps) one can earn for contributing to a project on Github.

Mostly crypto projects issue them as a fun way to earn some collectible and gamify contributing a bit. However there are [non-crypto OSS-projects](https://github.com/ActivityWatch/activitywatch/blob/master/CONTRIBUTING.md#claiming-gitpoap) giving them out aswell.

I think they're cool and could raise some awareness for our project in the crypto community, which is full of talented web devs passionate about open source.
Even cooler would be to hand them out to prompt taskers. They would have a public record of helping out of their very own, in addition to the planned leaderboard. But this could be significantly harder to implement due to lacking Github integration.

I can perfectly understand, if some of you think this would unnecessarily bloat the docs/this project.

Just throwing this out here, to see what you think :)

**Comment by fozziethebeat** [comment]
I'm triaging this as `nice-to-have`.  The core team probably won't be adding this but if someone wants to support this, please first describe what key integrations are needed (I assume we have to interact with wallets somehow, yes?) with a few checklist items.  Then feel free to provide PRs that work towards this goal.
